packages/react-dom/src/__tests__/ReactComponentLifeCycle-test.js

```diff
@@ -1065,16 +1065,20 @@ describe('ReactComponentLifeCycle', () => {
       }
     }
 
-    ReactTestUtils.renderIntoDocument(<Parent />);
+    const container = document.createElement('div');
+    document.body.appendChild(container);
+    ReactDOM.render(<Parent />, container);
     expect(divRef.current.textContent).toBe('remote:0, local:0');
 
     // Trigger setState() calls
     childInstance.updateState();
     expect(divRef.current.textContent).toBe('remote:1, local:1');
 
     // Trigger batched setState() calls
-    ReactTestUtils.Simulate.click(divRef.current);
+    divRef.current.click();
     expect(divRef.current.textContent).toBe('remote:2, local:2');
+
+    document.body.removeChild(container);
```

Comment:
We need to be careful with stuff like this because it won't clean up if the test fails. I tend to either use try/finally (annoying) or put it in beforeEach/afterEach.